platform/src/test/kotlin/org/stellar/anchor/platform/config/Sep10ConfigTest.kt

```diff
@@ -75,19 +78,44 @@ class Sep10ConfigTest {
     config.isClientAttributionRequired = true
     config.validateClientAttribution(errors)
     assertFalse(errors.hasErrors())
-    println(config.clientAttributionAllowList)
-    println(config.knownCustodialAccountList)
   }
 
   @Test
-  fun `test both client attribution deny and allow lists are empty`() {
-    // Create a cofnig with empty ClientsConfig lists
+  fun `test validation of empty client allow list when client attribution is required`() {
```

Comment:
Can we add tests for all the validations?

Reply:
Call `config.validate()` only tests the all validations are called in `config.validate()` function. If we think it is necessary, a test that `config.validate()` call all validation can be added. 

This is unit testing the `validateClientAttribution` function.

Reply:
If we still feel strong about this, we can add another test that `config.validate()` calls all validations in the future PR.